app/__init__.py

```diff
@@ -6,6 +6,25 @@
 app = Flask(__name__)
 
 
+
 @app.route('/')
 def index():
     return render_template('index.html', title="MLH Fellow", url=os.getenv("URL"))
```

Comment:
can you change it to

return render_template('/main/index.html', title="MLH Fellow", url=os.getenv("URL"))

^that is just because i had moved some files around, and made folders to make it more organized so now the path to index.html is what i wrote above